src/translations/en.json

```diff
@@ -1,38 +0,0 @@
-{
```

Comment:
Please, bring back the English translation. It's the main one.

Reply:
Totally by mistake, didn't notice. Sorry!